AuthenticationExample/AuthenticationExample/ProfileView.swift

```diff
@@ -73,6 +73,7 @@ struct ProfileView: View {
         isSigningOut = true
         Task {
             await ArcGISEnvironment.authenticationManager.revokeOAuthTokens()
+            await ArcGISEnvironment.authenticationManager.invalidateIAPCredentials()
             await ArcGISEnvironment.authenticationManager.clearCredentialStores()
```

Comment:
Maybe we need a "sign out" method on the authenticator now that we have critical mass of logic that needs to be done. If so, you can create an issue for that.

Reply:
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/issues/1183